test/framework/src/main/java/org/elasticsearch/test/ESTestCase.java

```diff
@@ -2178,6 +2179,12 @@ public static <T> T safeGet(Future<T> future) {
         }
     }
 
+    public static <T> Exception safeAwaitFailure(SubscribableListener<T> listener) {
+        return safeAwait(SubscribableListener.newForked(exceptionListener -> {
```

Comment:
Nit: unnecessary `{...}`, can just be an expression lambda

```suggestion
        return safeAwait(SubscribableListener.newForked(exceptionListener -> 
```